src/clients/inst_live_doit.rb

```diff
@@ -94,10 +95,9 @@ def main
     def LinksToCopyList
       cmd = Builtins.sformat(
```

Comment:
Now that the substitution is gone, calling `Builtins.sformat` is superfluous. And the lone `"\n"` can be removed too, it is an artifact of the conversion from YCP.